eng/AfterSigning.targets

```diff
@@ -29,5 +27,5 @@
   </Target>
 
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(GenerateChecksums)' == 'true'" />
```

Comment:
The checksums weren't produced without it, because SDK.targets imports https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Imports.targets, which imports `GenerateChecksums.targets`. `AfterSigning.targets` doesn't import that. Presumably publishing/signing.props don't need to import anything because they don't use custom tasks like `GenerateChecksums`

Reply:
Is there a way to directly import a specific `.targets` file from the Arcade SDK (@riarenas @JohnTortugo @chcosta)? If so that'd probably be best

Reply:
ping @riarenas @JohnTortugo @chcosta + @mmitche

Reply:
Also @jcagme